src/background/appManager.ts

```diff
@@ -170,7 +170,7 @@ export class AppManager {
 
     this.win.on('close', this.closeApp.bind(this))
 
-    if (DEVELOPMENT) {
+    if (!DEVELOPMENT) {
```

Comment:
```suggestion
```

Why not show it always?

Reply:
We decided to disable chrome shortcuts to avoid users opening the console or zooming in in order to give a native app feeling

Reply:
This code does not disable shortcuts, it enables the menu.

Reply:
It desables the shortcuts creating a new menu which overloads them with empty functions

Reply:
A comment about that would be nice.